sql/parser/builtins.go

```diff
@@ -761,3 +774,12 @@ func datumToRawString(datum Datum) (string, error) {
 
 	return "", fmt.Errorf("argument type unsupported: %s", datum.Type())
 }
+
+func round(x float64, n int64) (Datum, error) {
+	if n < 0 || n > 50 {
```

Comment:
50 seems about right.

* precision of `float64`s is about 17 decimal digits ([wikipedia](https://en.wikipedia.org/wiki/Double-precision_floating-point_format))
* the number of digits must be converted to `int`, therefore we need an upper bound
* rounding to more digits uses more resources (didn't check how it scales exactly)